src/test/rustdoc/structfields.rs

```diff
@@ -8,8 +8,6 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-// @has structfields/Foo.t.html
-// @has - struct.Foo.html
```

Comment:
Since this one actually checks the *content* of the redirect pages (and these lines were added in the PR that added the redirect pages in the first place) i feel like this test should also keep them.